Vagrantfile

```diff
@@ -67,7 +68,18 @@ else
     # cleanup openstack-kilo repo if required
     yum remove docker-engine -y || :
     yum-config-manager --disable openstack-kilo
-    curl https://get.docker.com | sed s/docker-engine/docker-engine-#{docker_version}/ | bash
+    if [[ #{docker_version} == *"rc"* ]]; then
+        echo "Getting pre-release docker version #{docker_version} "
+        curl -fsSL https://test.docker.com/ | sh
+    elif [[ #{docker_version} == *"cs"* ]]; then
+            echo "Getting cs docker version #{docker_version} "
+            curl -fsSL https://packages.docker.com/1.13/install.sh | sh
+    else
+        echo "Getting released docker version #{docker_version} "
+        curl https://get.docker.com | sed s/docker-engine/docker-engine-#{docker_version}/ | bash
+            echo "Getting released docker version #{docker_version} "
```

Comment:
location to fetch the rc (release candidate) and cs (commercial support) versions of docker are different from the released versions